.env.example

```diff
@@ -13,20 +14,27 @@ DB_USERNAME=homestead
 DB_PASSWORD=secret
 
 CACHE_DRIVER=redis
+FILESYSTEM_DRIVER=public
+QUEUE_CONNECTION=sync
```

Comment:
I went ahead and changed these based on what the framework now uses since I liked that they follow more closely to the filename and the array setting they are used in. I can make sure to update these on Heroku prior to deploy (add them before deploy, and remove the old named ones after successful deploy).

Reply:
We'll need to update these in our Terraform config [here](https://github.com/DoSomething/infrastructure/blob/246a5c8333226d50419a9affc50617084448431f/components/s3_bucket/main.tf#L203) and [here](https://github.com/DoSomething/infrastructure/blob/246a5c8333226d50419a9affc50617084448431f/components/sqs_queue/main.tf#L53). (We can start by adding the "new" ones alongside the old ones, and then remove the old ones after we've updated all of our apps to use the new environment variable names).

Reply:
I'm a little surprised by the `QUEUE_DRIVER` to `QUEUE_CONNECTION` change... it seems a little inconsistent with the "driver" terminology used elsewhere. I'm curious what the rationale (for Taylor & co.) was behind that!

Reply:
Ya, I wondered that myself... only explanation I could come up with is that with the "connections" config settings, each connection specifies its own `driver` so I suspect it would have been weird to specify a top level queue driver and then within another driver property, so:

```php
config('database.connections.pgsql.driver`);
```
vs

```php
config('database.driver.pgsql.driver`);
```

🤷‍♂️

Reply:
Ohhh, I bet that's it! :bulb: